docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,333 @@
+         q0_0 -> 2 ──────────────────────────────────────────────────────»
+                                                                         »
+    ancilla_0 -> 3 ──────────────────────────────────────────────────────»
+                                                                         »
+    ancilla_1 -> 4 ──────────────────────────────────────────────────────»
+                                                                         »
+    «                                                                     »
+    «     q0_1 -> 0 ───────────────────────────────────────────────────■──»
+    «               ┌──────────────┐┌────┐┌─────────┐┌───┐┌─────────┐┌─┴─┐»
+    «     q0_2 -> 1 ┤ Rz(-0.75949) ├┤ √X ├┤ Rz(π/4) ├┤ X ├┤ Rz(π/4) ├┤ X ├»
+    «               └──────────────┘└────┘└─────────┘└─┬─┘└─────────┘└───┘»
+    «     q0_0 -> 2 ───────────────────────────────────■──────────────────»
+    «                                                                     »
+    «ancilla_0 -> 3 ──────────────────────────────────────────────────────»
+    «                                                                     »
+    «ancilla_1 -> 4 ──────────────────────────────────────────────────────»
+    «                                                                     »
+    «               ┌─────────┐                                        ┌───┐     »
+    «     q0_1 -> 0 ┤ Rz(π/4) ├─────────────────────────────────────■──┤ X ├──■──»
+    «               ├─────────┴┐┌───┐┌──────────┐┌────┐┌─────────┐┌─┴─┐└─┬─┘┌─┴─┐»
+    «     q0_2 -> 1 ┤ Rz(-π/4) ├┤ X ├┤ Rz(3π/4) ├┤ √X ├┤ Rz(π/2) ├┤ X ├──■──┤ X ├»
+    «               └──────────┘└─┬─┘└──────────┘└────┘└─────────┘└───┘     └───┘»
+    «     q0_0 -> 2 ──────────────■──────────────────────────────────────────────»
+    «                                                                            »
+    «ancilla_0 -> 3 ─────────────────────────────────────────────────────────────»
+    «                                                                            »
+    «ancilla_1 -> 4 ─────────────────────────────────────────────────────────────»
+    «                                                                            »
+    «                                          
+    «     q0_1 -> 0 ────────────────────────■──
+    «               ┌───┐┌──────────┐┌───┐┌─┴─┐
+    «     q0_2 -> 1 ┤ X ├┤ Rz(-π/4) ├┤ X ├┤ X ├
+    «               └─┬─┘├─────────┬┘└─┬─┘└───┘
+    «     q0_0 -> 2 ──■──┤ Rz(π/4) ├───■───────
+    «                    └─────────┘           
+    «ancilla_0 -> 3 ───────────────────────────
+    «                                          
+    «ancilla_1 -> 4 ───────────────────────────
+    «                                          
+
+
+Plotting the Results
+====================
+
+You can visualize the results of your previous examples by generating a plot that show the depth, number of gates, and number of CX gates of your quantum circuits.
```

Comment:
Could we add a discussion of the results? At a first glance it looks strange that optimization level 3 has more gates -- we could for example point out that the target is to reduce the number of 2q gates, which is the best score for level 3.